Dockerfile

```
@@ -17,49 +10,166 @@
 #
 # Start with something like
 #
-# docker run -e 'DATABASE_URL=sqlite:////askbot-site/askbot.db' -e "SECRET_KEY=$(openssl rand 14 | base64)" -e ADMIN_PASSWORD=admin -p 8080:80 askbot/askbot:latest
+# docker run -e 'DATABASE_URL=sqlite:////askbot_site/askbot.sqlite' -e "SECRET_KEY=$(openssl rand 14 | base64)" -e ADMIN_PASSWORD=admin -p 8080:8000 askbot/askbot:latest
 #
-# User uploads are stored in **/askbot_site/askbot/upfiles** . I'd recommend to make it a kubernetes volume.
+# User uploads are stored in **/askbot_site/upfiles** . I'd recommend to make it a kubernetes volume.
+# Static files are stored in **/askbot_site/static** . I'd recommend to make it a kubernetes volume.
+
+# Stage 0
+FROM tiangolo/uwsgi-nginx:python3.8-alpine
```

```suggestion
FROM tiangolo/uwsgi-nginx:python3.9-alpine
```
3.8 would be downgrade